docs/en/observability/logs-monitor-datasets.asciidoc

```diff
@@ -1,14 +1,16 @@
 [[monitor-datasets]]
-= Monitor log data set quality
+= Monitor data set quality
```

Comment:
Maybe remove `Monitor` to align with other top-level items in the table of contents? 

```suggestion
= Data set quality
```